frontend/tests/dangerZone.spec.ts

```diff
@@ -0,0 +1,88 @@
+            if (realmType === "Regular") {
+                await page.locator("nav").getByRole("link", { name: parentPageName }).click();
+                await expect(page).toHaveURL("/moon");
+            }
+
+            // Create user realm
+            if (realmType === "User") {
+                await page.goto(`/@${userid}`);
+                await expect(page).toHaveURL(`/@${userid}`);
+            }
+
+            await page.getByRole("link", { name: "Page settings" }).click();
+        });
+
+        if (realmType === "Regular") {
+            await test.step("Path changing", async () => {
+                await test.step("Path can be changed", async () => {
+                    const pathInput = page.locator("input[name='pathSegment']");
+                    await pathInput.fill("new-path");
+                    await page.getByRole("button", { name: "Change path segment" }).click();
+
+                    await expect(page).toHaveURL("~manage/realm?path=%2Fnew-path");
+                });
+
+                await test.step("Links are updated", async () => {
+                    const links = [
+                        ["Go to page", "Far side"],
+                        ["Page settings", "Settings of page “Far side”"],
+                        ["Edit page content", "Edit page “Far side”"],
+                        ["Add subpage", "Add page"],
+                        ["Finanzamt", "Finanzamt"],
+                        ["Far side", "Far side"],
+                    ];
+
+                    const linkTest = async (page: Page, linkName: string, heading: string) => {
+                        await page.getByRole("link", { name: linkName }).first().click();
+                        await expect(page.getByRole("heading", { name: heading })).toBeVisible();
+                    };
+
+                    for (const [name, heading] of links) {
+                        await linkTest(page, name, heading);
+                    }
+                });
+            });
+        }
+
+
+        await test.step("Page can be deleted", async () => {
+            if (realmType === "User") {
+                await page.getByRole("button", { name: "Delete" }).click();
+                await page.getByRole("button", { name: "Delete" }).nth(1).click();
+
+                await expect(page.getByRole("heading", { name: "Tobira Videoportal" }))
+                    .toBeVisible();
+
+                await page.goto(`/@${userid}`);
+                await expect(page.getByRole("button", { name: "Create your own page" }))
+                    .toBeVisible();
+            } else {
+                await page.getByRole("link", { name: "Page settings" }).click();
+
+                await page.getByRole("button", { name: "Delete" }).click();
+                await page.getByRole("button", { name: "Delete" }).nth(1).click();
+
+                await expect(page.getByRole("link", { name: "Support page" }))
+                    .toBeHidden();
```

Comment:
How do we end up deleting the support page? I rather feel like the `if` block above, leaves us in some random page, which just so happens to make this pass. This whole delete step should probably navigate to a specific page first.

Reply:
Deleting the page (done in the lines above the ones you commented) should leave us on the homepage, though it wouldn't hurt to make sure that it does.
But I just realized that this shouldn't even test for "support page" as this uses a different realm now.
Now I wonder what made this pass before, because it shouldn't have (and in fact doesn't anymore, at least locally).